collision_detection_fcl/src/collision_common.cpp

```diff
@@ -455,6 +455,133 @@ bool distanceCallback(fcl::CollisionObject* o1, fcl::CollisionObject* o2, void*
   return cdata->done_;
 }
 
+
+bool distanceDetailedCallback(fcl::CollisionObject* o1, fcl::CollisionObject* o2, void* data, double& min_dist)
```

Comment:
This is OK for now.  Let's see what MoveIt thinks.  If it wasn't done before, then no reason to require doing it now.  Forcing everyone to check for NULL serves the greater purpose of discouraging use of raw pointers.